automl/cloud-client/deploy_model_test.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+import pytest
+
+import deploy_model
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+
+version_info = sys.version_info
+if version_info.major == 3:
+    if version_info.minor == 5:
+        MODEL_ID = "TEN5112482778553778176"
+    elif version_info.minor == 6:
+        MODEL_ID = "TCN3472481026502981088"
+    elif version_info.minor == 7:
+        MODEL_ID = "TST8532792392862639819"
+elif version_info.major == 2:
+    MODEL_ID = "TEN1499896588007374848"
+else:
+    MODEL_ID = "TEN7450981283112419328"
```

Comment:
Note: Experimentation to get tests pass when running 4 python versions in parallel.